L/LASzip/build_tarballs.jl

```diff
@@ -3,11 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "LASzip"
-version = v"3.4.3000"
+version = v"3.4.4"
```

Comment:
This version change won't sort correctly. You lowered the patch version from 3000 to 4, so the new build will look like an older version.

Reply:
Ah gotcha, I wasn't sure why it was set to 3000 rather than 3 but to make it match I'll set to 4000